src/themes/clean/templates/elements/journal/editorial_social_content.html

```diff
@@ -1,7 +1,9 @@
-<small>
+{% if user.website or user.twitter or user.facebook or user.github or user.linkedin %}
+<p><small>
 {% if user.website %}<a href="{{ user.website }}" aria-label="{{ member.user.full_name }}'s website" ><span class="fa fa-globe"> {% trans 'Website' %}</span></a>{% endif %}
 {% if user.twitter %}<a href="https://www.twitter.com/{{ user.twitter }}" aria-label="{{ member.user.full_name }}'s twitter profile"><span class="fa fa-twitter-square"> {% trans 'Twitter' %}</span></a>{% endif %}
 {% if user.facebook %}<a href="https://www.facebook.com/{{ user.facebook }}" aria-label="{{ member.user.full_name }}'s facebook profile"><span class="fa fa-facebook-square"> {% trans 'Facebook' %}</span></a>{% endif %}
 {% if user.github %}<a href="https://www.github.com/{{ user.github}}" aria-label="{{ member.user.full_name }}'s github profile"><span class="fa fa-github-square"> {% trans 'Github' %}</span></a>{% endif %}
 {% if user.linkedin %}<a href="https://www.linkedin.com/in/{{ user.linkedin }}" aria-label="{{ member.user.full_name }}'s linkedin profile"><span class="fa fa-linkedin-square"> {% trans 'Linkedin' %}</span></a>{% endif %}
```

Comment:
While you are at it, would you mind fixing the text being inside the span for font-awesome on each of these?
 It might have an effect on your change

Reply:
Fixed!